src/styles/variables.scss

```diff
@@ -0,0 +1,5 @@
+:root {
```

Comment:
You're using CSS variables instead of SCSS variables. In SCSS variables are defined using $ sign.